src/provider/non_hiding_zeromorph.rs

```diff
@@ -90,7 +90,7 @@ fn trim<E: Engine>(
 }
 
 /// Commitments
-#[derive(Debug, Clone, Eq, PartialEq, Default, Serialize, Deserialize)]
+#[derive(Debug, Eq, PartialEq, Default, Serialize, Deserialize)]
```

Comment:
Given that `E::G1Affine` implements `Copy` I think we can actually keep this one.